lib/datadog/tracing/contrib/aws/services/dynamodb.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
```

Comment:
For namespacing, in Ruby, it's standard practice to use singular names for folders, so I recommend renaming this directory from `aws/services` to `aws/service`.